ThunderCloud/VideoPlayerControlsView.swift

```diff
@@ -45,10 +45,20 @@ open class VideoPlayerControlsView: UIView {
 	public required init?(coder aDecoder: NSCoder) {
 		super.init(coder: aDecoder)
 	}
+    
+    private func getBottomSafeAreaInset() -> CGFloat {
```

Comment:
Again, if our deployment target has been bumped, don't need this check :) Also again: re-indent :P